package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@unlok-co/nuxt-stripe",
-  "version": "1.1.1",
+  "version": "1.3.0",
```

In general I'd advise against incrementing a version number within the PR.

```suggestion
  "version": "1.1.1",
```